services/app-api/src/resolvers/questionAnswers/indexQuestions.test.ts

```diff
@@ -0,0 +1,209 @@
+import INDEX_QUESTIONS from 'app-graphql/src/queries/indexQuestions.graphql'
+import CREATE_QUESTION from 'app-graphql/src/mutations/createQuestion.graphql'
+import {
+    constructTestPostgresServer,
+    createAndSubmitTestHealthPlanPackage,
+} from '../../testHelpers/gqlHelpers'
+import { UserType } from '../../domain-models'
+
+describe('indexQuestions', () => {
+    const testUserCMS: UserType = {
+        id: 'f7571910-ef02-427d-bae3-3e945e20e59d',
+        role: 'CMS_USER',
+        email: 'zuko@example.com',
+        familyName: 'Zuko',
+        givenName: 'Prince',
+        stateAssignments: [],
+    }
+    it('returns package with questions', async () => {
+        const server = await constructTestPostgresServer()
+        const cmsServer = await constructTestPostgresServer({
+            context: {
+                user: testUserCMS,
+            },
+        })
+
+        const submittedPkg = await createAndSubmitTestHealthPlanPackage(server)
+
+        await cmsServer.executeOperation({
```

Comment:
You could create a helper function for creating questions like we did for setting up the packages. For test helper functions it’s OK if they throw an error if something goes wrong since that will end the test ASAP